content/en/docs/guides/packaging-an-application.md

```diff
@@ -277,6 +352,8 @@ metadata:
 #...
 ```
 
+Validate the package again:
+
```

Comment:
I did not see the below error. Instead, I saw: 
$ kubectl package validate 1_applications/2_templates/
Error: validating package: loading package from files: convert files to package: Package validation errors:
 PackageManifest invalid in manifest.yaml:
 test.template[0].context.config.nginxImage: Required value

Then I added 
  ```
      config:
        nginxImage: "nginx:1.23.3"   
```  
to the manifest.yaml file and ran the validate command again, but still saw a different error than indicated in documentation:
`Error: validating package: loading package from files: validate package and files: executing template from deployment.yaml.gotmpl with context map[config:map[nginxImage:nginx:1.23.3] environment:map[kubernetes:map[version:]] images:map[] package:map[metadata:map[annotations:map[] labels:map[] name: namespace:]]]: template: :4:19: executing "" at <.package.metadata.Name>: map has no entry for key "Name"`


Should the example manifest file here: https://github.com/package-operator/examples/blob/main/1_applications/2_templates/manifest.yaml   also contain 
       ```
 config:
        nginxImage: "nginx:1.23.3"     
```  
or would you expect a new user to just know they should add that?

Reply:
No idea honestly, anybody else? @thetechnick @erdii?